snakes_cafe.py

```diff
@@ -51,4 +56,4 @@
         print(f' ** {list[order]} order has been added to your order **')
```

Comment:
You are not confirming that the item the user is ordering is available as a menu item. This is most easily done by having your menus set up as a list of dictionaries, which you then iterate over to see whether or not the user's entry matches a key in any of the dictionaries.